<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Machine</class>
 <widget class="QMainWindow" name="Machine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Machine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Machine              </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Machine_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Description         </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Description"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Alias                    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Alias"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>PurchaseDate    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="purchaseDate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>MachineGroup   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MachineGroup"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>MachineEnable   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MachineEnable"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>DateCreated        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DateCreated"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>CreatedBy            </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CreatedBy"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>DatelastModified</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DatelastModified"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>LastModifiedBy   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LastModifiedBy"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
